### ISSUE TITLE
Deleting src account does not update UI [qubes]

### ISSUE THREAD

**Issue by creviera** [freedomofpress/securedrop-client#545]
## Description

Deleting a source account from the client on qubes does not make the source disappear from the sourcelist until you close the client and reopen. Also seeing this warning in stdout:

```
SAWarning: DELETE statement on table 'messages' expected to delete 2 row(s); 0 were matched. Please set confirm_deleted_rows=False within the ampper configuration to prevent this warning.
```

**Comment by deeplow** [comment]
This is likely a duplicate of #360. I think I may have diagnosed that bug and will post that in #360. But I was not able to reproduce the part of it not updating the UI in Qubes. For me the source is successfully deleted in the interface as well as in the the database, but the warning is still there. Was it only in Qubes that this happened or your didn't test in other system?

**Comment by deeplow** [comment]
@creviera I was not able to reproduce the part where the qubes UI was not updating, but when you have time, take a look at this to see if still happens. #581 closed #360 which seems similar to this one

**Comment by redshiftzero** [comment]
resolved as of 0.1.3
